vip-helpers/vip-utils.php

```diff
@@ -1539,6 +1539,7 @@ function vip_is_jetpack_request() {
  * @param $interval integer Interval in seconds between two messages sent from one DC
  */
 function wpcom_vip_irc( $channel_or_user, $message, $level = 0, $kind = '', $interval = 0 ) {
+	_deprecated_function( __FUNCTION__, '', 'Alerts::chat' );
```

Comment:
Just noting that this will throw `E_USER_DEPRECATED` by default, which we won't catch in logstash.

So prior to actually removing the method, we'll probably need to swap it to trigger_error() with a warning-level to ensure it's not actually used anywhere else (I'm quite sure it is).